### ISSUE TITLE
Suggestion: search twitter account descriptions & pinned tweet for bsky/bluesky

### ISSUE THREAD

**Issue by tdbe** [kawamataryo/sky-follower-bridge#9]
I only looked at the code quickly/lazily, but as far as I understand, you search for the twitter @handle (and then the display name) to see if it also comes up in bluesky. Sometimes people write their bluesky in their description or url or location or pinned tweet. And it can be different than their twitter handle.

Perhaps just highlight the account if you find "bsky" or "bluesky" anywhere in those fields and let the human figure it out manually. 🤷‍♂️

**Comment by cooljeanius** [comment]
yeah the way @lucahammer did fedifinder is worth comparing against

**Comment by kawamataryo** [comment]
提案ありがとう。ただ、このツールではfollowingのユーザーリストに表示されている情報のみが、検出対象です。固定ツイートの重宝をみることはできません。
